<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numerical</class>
 <widget class="QWidget" name="Numerical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>算术计算</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pb_cal">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="le_input"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="l_result">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
